orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -160,37 +162,59 @@ def SendViaFTPS(resourceId: str) -> None:
     Makes a POST API call to upload the resource to an FTPS server
     using orthanc credentials as authorisation
     """
-    # Query orthanc-anon for the study
+    # Download zip archive of the DICOM resource
     query = f"{ORTHANC_URL}/studies/{resourceId}/archive"
+    fail_msg = "Could not download archive of resource '%s'"
+    response_study = _make_query(resourceId, query, fail_msg)
+
+    # get the zip content
+    zip_content = response_study.content
+    logging.info("Downloaded data for resource %s", resourceId)
+
+    upload.upload_dicom_image(zip_content, GetPatientID(resourceId))
+    logging.info("Uploaded data to FTPS for resource %s", resourceId)
+
+
+def GetPatientID(resourceId: str) -> str:
+    """
+    Queries the Orthanc instance to get the PatientID for a given resource.
+    When anonymisation has been applied, the PatientID is the pseudo-anonymised ID.
+    """
+    query = f"{ORTHANC_URL}/studies/{resourceId}"
```

Comment:
I think its cleaner to use the suggestion from #190 `{dicom-server}/instances/{resource-id}/content/0010-0020` so we just query the tag directly rather than downloading all of the dicom tags

Reply:
Aaah but this doesn't actually work with the `resourceID` 😭 
AFAIK `resourceID` can only be used for `/studies/` queries 😕

Reply:
I've looked into it a bit more and I don't see a way to get to the specific tag that isn't more cumbersome then the current query I'm doing, so think I will stick with that.